marquez_client/client.py

```diff
@@ -29,10 +29,15 @@
 _USER_AGENT = f'marquez-python/{VERSION}'
 _HEADERS = {'User-Agent': _USER_AGENT}
 
+_NAMESPACE_LENGTH_LIMIT = 1024
+_SOURCE_LENGTH_LIMIT = 1024
+_DATASET_LENGTH_LIMIT = 1024
+_JOB_LENGTH_LIMIT = 1024
 
 class MarquezClient(object):
-    def __init__(self, host=None, port=None,
+    def __init__(self, protocol=None, host=None, port=None,
```

Comment:
Nice! Adding an option for setting the `protocol` is a great idea 💯

Reply:
_Minor:_ The client would only support `https` and `http`. We can rename `protocol` to `ssl` and define it as a flag:

```python
class MarquezClient(object):
  def __init__(self, ssl=False, host=None, port=None, ...
```

And depending on the value of `ssl`, we can add `https` or `http` with the appropriate ports

Reply:
done.